### PR TITLE
Removed deprecated article and mentions of it

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
CMS v 14 and 15

## Deadline (if relevant)

Anytime

### CODE REVIEW

**Review by sofietoft — APPROVED** [review]
Looks good!
Remember redirects when you delete an article 💪 

Might make sense to link to the Umbraco Flavoured Markdown here 🤔

**Comment by eshanrnh** [comment]
Yup, I've added redirects to the Umbraco flavored markdown article.

**Comment by sofietoft** [comment]
Great! When we make these kind of redirects, does it make sense to add some information to the article we're redirecting to?
To clear up any confusion that might occur if you're confused about seeing a different article that the one you where looking for? 🤔

**Comment by eshanrnh** [comment]
That's a great point 🙌 It will definitely make sense to provide some context on the redirected article. I have added a Note in the article. Let me know what you think 😀

File: 15/umbraco-cms/reference/umbraco-flavored-markdown.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Umbraco Flavored Markdown
 
+{% hint style="info" %}
+This article also addresses the Label Property Configuration. If you were looking for that content, you're in the right place.
```
Comment (sofietoft) [inline 15/umbraco-cms/reference/umbraco-flavored-markdown.md:4]: In other places where we've done something similar (like the [Content Picker article](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker)), we ask a question.
So, maybe for consistency we do that here as well?

"Are you looking for Label Property Configuration?
"Umbraco Flavoured Markdown replaces Label Property Configuration in Umbraco 14+."

**Comment by eshanrnh** [comment]
I've tweaked the Note slightly to mention why we are using UFM 😀Hope it's fine.

**Review by sofietoft — APPROVED** [review]
Looks great! Super idea adding a bit more detail to the note 🙌

**Pull request merged**